apps/election-manager/src/utils/Storage.ts

```diff
@@ -1,27 +1,24 @@
 import { strict as assert } from 'assert'
 
-// TODO: define a type that's actually serializable
-export type Serializable = unknown
-
 /**
```

Comment:
are you planning to eventually move this file to some shared directory since it seems duplicated in bmd and election-manager right now?

Reply:
Probably, but it wasn't clear there was benefit in doing it now especially with the shortcomings mentioned in the description.